docs/book/src/watching-resources/externally-managed.md

```diff
@@ -0,0 +1 @@
+# Externally Managed Resources
```

Comment:
Is this file needed?

Reply:
So I split out these files in order to make space for the information [requested here](https://github.com/kubernetes-sigs/kubebuilder/pull/2000#discussion_r573860455).

I'm perfectly happy to get this working with one page of documentation, then split it up and add more at a later date, in a separate PR. Will defer to y'alls guidance here.

Reply:
Scratch this, didn't notice that these were empty files that should not have existed.

Reply:
Just to make it clear. I was just refering to these empty files not to the fact that you are using 3 files for the documentation. I'm totally cool with that.